### DISCUSSION TITLE
Hello anyone can help me ORM Where Associated Conditions

### DISCUSSION THREAD

**Discussion by aiqbalsyah** [loopbackio/loopback-next#10499] [Q&A]
It will be nice if i know to solve this code.

![carbon(1)](https://github.com/loopbackio/loopback-next/assets/22832751/215a400b-9841-4fa9-a26a-4272de079a32)

**Comment by Salketer** [comment]
This cannot be done like that. What you'd do in this case is find all clubs which name matches, get their ID and then query the matches where homeClubId:{inq:matchingClubIds}

**Comment by aiqbalsyah** [comment]
sorry i forgot to solve this. is already solved using sequelize 
`$homeClub.name$` it solved
